src/server/scripts/Outland/Auchindoun/ManaTombs/boss_pandemonius.cpp

```diff
@@ -94,7 +94,7 @@ class boss_pandemonius : public CreatureScript
                     else
                     {
                         events.ScheduleEvent(EVENT_VOID_BLAST, 500ms);
-                        events.DelayEvents(EVENT_DARK_SHELL, 500);
+                        events.DelayEvents(500ms, EVENT_DARK_SHELL);
```

Comment:
Type-safety FTW. Although I must admit, the call itself doesn't make much since `EVENT_DARK_SHELL` isn't assigned to a group...